crates/autopilot/src/shadow.rs

```diff
@@ -79,9 +74,7 @@ impl RunLoop {
     pub async fn run_forever(mut self) -> ! {
         let mut previous = None;
         loop {
-            if let RunLoopMode::SyncToBlockchain = self.synchronization {
-                let _ = ethrpc::block_stream::next_block(&self.current_block).await;
-            };
+            let _ = ethrpc::block_stream::next_block(&self.current_block).await;
```

Comment:
Small comment explaining that we use this as a synchronization mechanism would be nice.